libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -563,7 +564,8 @@ bool ASTJsonConverter::visit(InlineAssembly const& _node)
 
 	Json::Value externalReferencesJson = Json::arrayValue;
 
-	for (auto&& it: boost::range::sort(externalReferences))
+	ranges::sort(externalReferences);
+	for (auto&& it: externalReferences)
 		externalReferencesJson.append(std::move(it.second));
```

Comment:
I think this could be refactored using `values`:
```suggestion
	for (Json::Value& identifier: externalReferences | ranges::views::values)
		externalReferencesJson.append(std::move(identifier));
```

Reply:
done